css/style.css

```diff
@@ -17,3 +16,227 @@ body {
  */
 
 
+
+/** navigation */
+nav {
+  background: white;
+  height: 60%;
+  width: 100%;
+}
+
+.nav-position {
+  display: flex;
+  justify-content: space-between;
+}
+.nav-logo {
+  width: 40px;
+  height: auto;
+  margin: 10px 45px;
+}
+
+.nav-ul {
+  float: right;
+  margin-right: 20px;
+  padding: 10px;
+  display: flex;
+  justify-content: space-between;
+}
+
+nav ul li {
+  display: inline-block;
+  line-height: 60px;
+  margin: 10px;
+  font-size: 18px;
+  color: darkgrey;
+}
+
+nav ul li a {
+  color: rgb(15, 15, 15);
+  font-size: 20px;
+  padding: 5px, 12px;
+  border-radius: 3px;
+}
+
+a.active, a:hover {
+  color: orangered;
+  transition: 0.5s;
+}
+
+.hamburger {
+  display: none;
+}
+@media (min-width: 400px) and (max-width: 767px) {
+  nav-logo {
+    padding-left: 50px;
+  }
+  nav ul li a {
+    font-size: 16px;
+  }
+  .hamburger {
+    display: inline-block;
+  }
+
+  .nav-ul {
+    display: none;
+  }
+
+  .navul-a {
+    text-decoration: none;
+    color: black;
+  }
+
+  footer li {
+    padding-bottom: 1rem;
+
+  }
+
+  .contact-ul {
+    flex-direction: column;
+    align-items: center;
+    
+  }
+}
+@media (min-width:768px) and (max-width: 900px)
+
+
+/**--page main background section*/
+/**.img-background {
+  width: 100%;
+  height: 100vh;
+}*/
+
+.main-devices{
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: space-around;
+  text-align: end;
+}
+
+.learn {
+  margin: 0;
+  background-color: orangered;
+  border-radius: 5px;
+  width: 120px;
+  line-height: 20px;
+  padding: 10px;
+  font-size: 18px;
+}
+
+.background {
+  background: url("../img/first-background.jpg") no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 70vh;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  font-size: 40px;
+}
+
+.img-main h1 {
```

Comment:
Instead of having several `h1` tags and giving them different font-sizes in CSS, it would be best to make use of the other heading tags available to you such as. `h2`, `h3` etc